src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -43,14 +46,31 @@ class HelpWebViewController: UIViewController, UIWebViewDelegate, UIScrollViewDe
 
     deinit {
         url = nil
-        webView.delegate = nil
+        webView?.uiDelegate = nil
         webView = nil
     }
 
     // MARK: - View Lifecycle
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        let webConfiguration = WKWebViewConfiguration()
+        webConfiguration.allowsInlineMediaPlayback = true
+
+        webView = WKWebView(frame: self.viewForWebView.bounds, configuration: webConfiguration)
+        webView?.uiDelegate = self
+        webView?.navigationDelegate = self
+        webView?.translatesAutoresizingMaskIntoConstraints = false
+        if webView != nil {
+            viewForWebView.addSubview(webView!)
```

Comment:
This will still produce a linter warnings because of force unwrapping. Use `if let` instead (e.g. `if let webView = webView`).